packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1448,30 +1666,38 @@ class DiagnosticsProperty<T> extends DiagnosticsNode {
   /// Use if computing the property [value] may throw an exception or is
   /// expensive.
   ///
-  /// The [hidden], [showName], [showSeparator], and [style] arguments must not
-  /// be null.
+  /// The [showName], [showSeparator], [style], and [missingIfNull] arguments
+  /// must not be null.
+  ///
+  /// At most one of `hidden` and [level] should be specified. Setting `hidden`
+  /// to `true` is equivalent to setting [level] to [DiagnosticLevel.hidden].
+  /// If neither `hidden` or [level] is set, [level] defaults to
+  /// [DiagnosticsLevel.info].
   DiagnosticsProperty.lazy(
```

Comment:
these docs should talk about how the "level" argument is just a suggestion and how it can be overridden by other things.

Reply:
done